cluster-autoscaler/cloudprovider/openshiftmachineapi/machineapi_nodegroup_test.go

```diff
@@ -314,48 +299,33 @@ func TestNodeGroupIncreaseSize(t *testing.T) {
 		for i, tc := range testCases {
 			t.Run(tc.description, func(t *testing.T) {
 				annotations := map[string]string{
-					nodeGroupMinSizeAnnotationKey: tc.minSize,
-					nodeGroupMaxSizeAnnotationKey: tc.maxSize,
+					nodeGroupMinSizeAnnotationKey: "1",
```

Comment:
what's the reason for harcoding now?

Reply:
It helps validates the inputs.

Reply:
> what's the reason for harcoding now?

And my correct answer is...

They were identical for each test case so I reduced the number of fields in the `type testCase` with the intent of making things simpler.